ui-v2/config/environment.js

```diff
@@ -65,24 +65,32 @@ module.exports = function(environment) {
       }
       return 'oss';
     })(),
-    CONSUL_ACLS_ENABLED: true,
-    CONSUL_NSPACES_ENABLED: true,
-    CONSUL_NSPACES_UNDEFINED_NAME: 'default',
+    CONSUL_ACLS_ENABLED: false,
+    CONSUL_NSPACES_ENABLED: false,
     CONSUL_HOME_URL: 'https://www.consul.io',
     CONSUL_DOCS_URL: 'https://www.consul.io/docs',
     CONSUL_DOCS_LEARN_URL: 'https://learn.hashicorp.com/consul',
     CONSUL_DOCS_API_URL: 'https://www.consul.io/api',
     CONSUL_COPYRIGHT_URL: 'https://www.hashicorp.com',
     CONSUL_COPYRIGHT_YEAR: '2019',
   });
-  if (environment === 'development') {
-    // ENV.APP.LOG_RESOLVER = true;
-    // ENV.APP.LOG_ACTIVE_GENERATION = true;
-    // ENV.APP.LOG_TRANSITIONS = true;
-    // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
-    // ENV.APP.LOG_VIEW_LOOKUPS = true;
+  switch (true) {
```

Comment:
don't think I've seen this before, feels a little tricky though for just two cases though

Reply:
We'll probably move all the conditional forking here to use the switch statement then we can remove our `isDevLike`/`isProdLike` and just use:

```javascript
case 'development':
case 'staging':
case 'test':
  // isDevLike stuff
  break;
case 'test':
  // test only
  break;
```
So this will grow to more than 2, just I didn't want to change too much here.